ci-operator/jobs/openshift/knative-client/openshift-knative-client-release-next-presubmits.yaml

```diff
@@ -0,0 +1,66 @@
+presubmits:
```

Comment:
you did generate those, running the latest prow docker image?

Reply:
Yup, just rechecked there is `:latest` tag